packages/react/src/components/SignOutButton.tsx

```diff
@@ -1,25 +1,29 @@
-import type { SignOutCallback, SignOutOptions } from '@clerk/types';
+import type { SignOutOptions } from '@clerk/types';
 import React from 'react';
 
 import type { WithClerkProp } from '../types';
 import { assertSingleChild, normalizeWithDefaultValue, safeExecute } from '../utils';
 import { withClerk } from './withClerk';
 
 export type SignOutButtonProps = {
-  signOutCallback?: SignOutCallback;
+  redirectUrl?: string;
   signOutOptions?: SignOutOptions;
   children?: React.ReactNode;
 };
 
 export const SignOutButton = withClerk(
   ({ clerk, children, ...props }: React.PropsWithChildren<WithClerkProp<SignOutButtonProps>>) => {
-    const { signOutCallback, signOutOptions, ...rest } = props;
+    const { redirectUrl = '/', signOutOptions, ...rest } = props;
 
     children = normalizeWithDefaultValue(children, 'Sign out');
     const child = assertSingleChild(children)('SignOutButton');
 
+    const navigateToRedirectUrl = () => {
+      return clerk.navigate(redirectUrl);
```

Comment:
🤔  should we use this or `clerk.navigate(redirectUrl, { replace: true })`?

Reply:
Inside the `navigate` function:

```ts
const customNavigate =
  options?.replace && this.#options.routerReplace ? this.#options.routerReplace : this.#options.routerPush;
```

So I don't think this would do anything, right?

Reply:
Ok, so this refers to push and replace in regards to the browser history. I'd want to push here, so code is fine as-is 👍